xtask/src/commands/release/bump_version.rs

```diff
@@ -53,8 +53,14 @@ pub fn bump_version(workspace: &Path, args: BumpVersionArgs) -> Result<()> {
                 Version::Patch => VersionBump::Patch,
             }
         };
-        let mut package = CargoToml::new(workspace, package)?;
-        update_package(&mut package, &version, false)?;
+        let mut pkg = CargoToml::new(workspace, package)?;
```

Comment:
This isn't the only place we call `update_package`, I suggest we do the check in there instead.